### DISCUSSION TITLE
1D Plane wave Propagation through anisotropic materials?

### DISCUSSION THREAD

**Discussion by jaredmagnusson** [thliebig/openEMS-Project#27] [General]
Hello, does openEMS have the capability of modeling a 1D plane wave through a material with spatially+frequency varying, complex permittivity? I'm interested in calculating attenuation/reflection/transmission through a 1D plasma slab. How could I best go about this? Or are their other codes better suited to this? I'm very new to the world of EM and signals, I apologize if this question is too obvious

Thank you.

**Comment by thliebig** [comment]
Well openEMS can do an-isotropic (but only main axis) materials of pretty much all kinds. But it has to be linear.
Setting up a plane wave is possible too. I'm not sure what you mean with 1D plane wave? Maybe you mean a fully guided wave that only depends on one coordinate? That would be possible too using PEC and PMC boundary conditions.
